### PR TITLE
start binning docs

### PR DESCRIPTION
**Pull request by cyrush**
@mclarsen created a PR so I can easily share a few suggestions.

### CODE REVIEW

File: src/docs/sphinx/Actions/Binning.rst [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+(sum, average, min, max, variance, etc) depending on the type of analysis desired.
+
+You can bin spatially to calculate distributions, find extreme values, etc.
+With the right approach, you can implement mesh agnostic analysis that can be used across simulation codes.
+You can also map the binned result back onto the original mesh topology
+to enable further analysis, like deviations from an average.
+
+Benefits
+--------
+Simulation user often needs to analyze quantaties of interest within fields on
```
Comment (cyrush) [inline src/docs/sphinx/Actions/Binning.rst:63]: users often need

File: src/docs/sphinx/Actions/Binning.rst [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+a mesh, but the user might not know the exact data structures used by the underlying
+simulation.
+For example, the mesh data might be represented as uniform grids or as high-order finite
+element meshes.
+If the users does not know the underlying data structures, it can be very difficult to write
```
Comment (cyrush) [inline src/docs/sphinx/Actions/Binning.rst:68]: if users do not know

File: src/docs/sphinx/Actions/Binning.rst [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+the underlying analysis, and that analysis code will not work for another simulation.
+Using spatial binning essentially create a uniform representation that can be use across
+simulation codes, regardless of the underlying mesh representation.
+
+
+Sampling and Aggregation
+------------------------
+When specifying the number of bins on an axis, there will always be over smapling or undersampling.
+During spatial binning, each zone is placed into a bin based on it centriod, and with all
```
Comment (cyrush) [inline src/docs/sphinx/Actions/Binning.rst]: and as with all

File: src/docs/sphinx/Actions/Binning.rst [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+.. figure:: ../images/undersampling.png
+
+  An example of spatial under sampling.
+
+
+.. figure:: ../images/oversampling.png
+
+  An example of spatial over sampling.
+
+When multiple values fall into a sinlge bin, we aggregate values using the following options:
```
Comment (cyrush) [inline src/docs/sphinx/Actions/Binning.rst]: single

**Review by cyrush — COMMENTED** [review]
looks good!

File: src/docs/sphinx/Actions/Binning.rst (hunk 3 above)
Comment (mclarsen) [reply src/docs/sphinx/Actions/Binning.rst]: I disagree

**Pull request merged**